src/amp.js

```diff
@@ -52,19 +52,6 @@ import {preconnectToOrigin} from './preconnect';
 import {startupChunk} from './chunk';
 import {stubElementsForDoc} from './service/custom-element-registry';
 
-/**
- * self.IS_AMP_ALT (is AMP alternative binary) is undefined by default in the
- * main v0.js since it is the "main" js.
- * This global boolean is set by alternative binaries like amp-inabox and
- * amp-shadow which has their own bootstrapping sequence.
- * With how single pass works these alternative binaries cannot be generated
- * easily because we can only do a "single pass" so we treat these alternative
- * main binaries as "extensions" and we concatenate their code with the main
- * v0.js code.
- * @type {boolean|undefined}
- */
-const shouldMainBootstrapRun = !self.IS_AMP_ALT;
```

Comment:
According to the description, don't we still need this variable for `amp-inabox` and `amp-shadow`?

Reply:
sorry late reply. this should have been only needed for single pass, if theres any other description i haven't removed about it, could you point it out? thanks